packages/synapse-react-client/src/components/DownloadCart/DownloadCartPage.tsx

```diff
@@ -72,7 +72,7 @@ export const DownloadCartPage: React.FunctionComponent<
           <div className="grid">
             <h3 className="pageHeaderTitle">Your Download Cart</h3>
             <Tooltip
-              title="Immediately removes all items from your download list"
+              title="Immediately removes all items from your download cart"
```

Comment:
Fix couple places where "download list" was used inconsistent with the rest of the copy